### ISSUE TITLE
dom-event repository not found: use npm instead

### ISSUE THREAD

**Issue by BaptisteBriel** [bigwheel-framework/bigwheel#7]
Hi,

I had an issue while installing bigwheel on npm:
The "dom-event" is not on the repository (https://github.com/bigwheel-framework)

That makes an error on `npm i` or `npm install bigwheel`: repository not found

```json
"dependencies": {
    "bw-router": "^1.1.1",
    "bw-viewmediator": "^2.0.0",
    "bw-vm": "^2.0.0",
    "dom-event": "git://github.com/bigwheel-framework/dom-event"
},
```

Maybe using the npm version would be easier?

```json
"dependencies": {
    "bw-router": "^1.1.1",
    "bw-viewmediator": "^2.0.0",
    "bw-vm": "^2.0.0",
    "dom-event": "0.0.4"
},
```

Let me know if I did something wrong. :)
Otherwise, it works like a charm. Awesome work!

Thanks!

**Comment by MikkoH** [comment]
Crap thats my bad. That repo exists but it's at http://github.com/mikkoh/dom-event.

That's my bad. I had migrated bigwheel from my user account mikkoh to the org bigwheel-framework but had forgot to update the related `package.json` fields. So I just simply went in brute force and did a find and replace for mikkoh - > bigwheel-framework. I forgot I had that one dep that relied on a special version of `dom-event`.

The reality is I could probably just simply use `dom-event` the only difference is that a `NodeList` is returned if the query string matches many things. I believe in `bigwheel`s case it's fine to use default `dom-event`. I'll investigate. 

https://github.com/npm-dom/dom-event/pull/4

Thanks for bringing this up.
